src/confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -159,10 +173,17 @@ def _send_request(self, url, method='GET', body=None, headers={}):
             raise ClientError("Method {} is invalid; valid methods include {}".format(method, VALID_METHODS))
 
         _headers = {'Accept': ACCEPT_HDR}
+        nbody = body
         if body:
-            _headers["Content-Length"] = str(len(body))
-            _headers["Content-Type"] = "application/vnd.schemaregistry.v1+json"
```

Comment:
Is content type header not required anymore?

Reply:
the constant have this already declared and the constant was declared before also